rpc/block.go

```diff
@@ -8,25 +8,25 @@ import (
 	"github.com/NethermindEth/juno/core/felt"
 )
 
-// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_api_openrpc.json#L1999-L2008
-type Status uint8
```

Comment:
Please make sure each individual commit compiles fine and all tests pass. There are still place we use `Status` in this commit.

Unless you want this to be squash merged tho, in that case it is fine.

Reply:
Yikes, sorry about this one. Not sure what I was thinking when I made this PR. Should be fixed now.